spec/models/comment_spec.rb

```diff
@@ -0,0 +1,16 @@
+# spec/models/comment_spec.rb
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  let(:user) { create(:user) }
+  let(:post) { create(:post) }
+
+  describe 'validations' do
+    it { should validate_presence_of(:text) }
+  end
+
+  describe 'associations' do
+    it { should belong_to(:user) }
+    it { should belong_to(:post) }
+  end
+end
```

Comment:
- [ ] Kindly add test for the `increment_post_comments_counter` method in the Comment model file.